geode-docs/reference/statistics_list.html.md.erb

```diff
@@ -553,7 +553,12 @@ These statistics are in a client and they describe one of the client’s connect
 
 ## <a id="section_66C0E7748501480B85209D57D24256D5" class="no-quick-link"></a>Continuous Querying (CqQueryStats)
 
-These statistics are for continuous querying information. The statistics are:
+These statistics are for continuous querying information.
+
+**Note:**
+The subscription redundancy changes CQS_ACTIVE, CQS_STOPPED and CQS_CLOSED count on partitioned region
```

Comment:
May we discuss what is intended in this Note? I don't understand it, so I'd like to help revise the wording?

Reply:
Hi @karensmolermiller,
After talking with @agingade , we found solution for this issue, but there must be added this note to inform users that changing subscription redundancy level on partitioned region will have effect on these counters as they will change depending on subscription redundancy level.

Reply:
Ok.  Then, the prose belongs in the description next to each of these 3 statistics.  And, that prose ought to say what the effect is (and not just that the sub. red. level affects the values of the counters). If you'd like to talk directly about wording, let me know and we'll find a way.